containers/node-bootstrap-container/entrypoint.sh

```diff
@@ -25,4 +25,5 @@ cd $WORKSPACE_DIR/$MONO_REPO_NAME/containers/node-bootstrap-container
 FOLDER_NAME=$(node create-folder-name.js $API_ID) || node create-gh-issue.js
 mkdir -p "$WORKSPACE_DIR/$MONO_REPO_NAME/packages/$FOLDER_NAME" || node create-gh-issue.js
 cd  $WORKSPACE_DIR/$MONO_REPO_NAME
+export _OWLBOT_YAML_PATH="packages/$FOLDER_NAME/.OwlBot.yaml"
```

Comment:
Where is this env var being used? Should it be an argument to the gapic-node-templating CLI instead?